### PR TITLE
Add tools/pocolog2msgpack

### PR DESCRIPTION
**Pull request by AlexanderFabisch**
@2maz Could you check if I specified everything correctly? Do I also have to specify the optional Python packages?

### CODE REVIEW

**Comment by 2maz** [comment]
If the python packages are not yet defined in an osdeps file, then pls add them.
Otherwise specification looks ok.

File: rock.osdeps [hunk 1]
```diff
@@ -251,3 +251,18 @@ ruby-backports:
 ruby-xmlrpc:
    gem: xmlrpc
 
+python:
```
Comment (AlexanderFabisch) [inline rock.osdeps]: Not sure if Python has to be defined or whether it is already known to autoproj.

Comment (2maz) [reply rock.osdeps]: Currently it is defined in rock package_set. We can remove it there, once the osdeps has been defined in rock.core.

rock/rock.osdeps
```
python:
    debian, ubuntu: python-dev
    opensuse: python-devel
```

Comment (AlexanderFabisch) [reply rock.osdeps]: OK, should be fixed now...

**Comment by AlexanderFabisch** [comment]
Yep, here: https://github.com/rock-core/rock-package_set/pull/127

**Comment by 2maz** [comment]
Ok, I think you meant rock-core/rock-package_set#182
Thanks, anyhow.

**Comment by AlexanderFabisch** [comment]
Yes, I don't know how that happened. 😀

**Pull request merged**